library/translation.inc.php

```diff
@@ -263,6 +263,15 @@ function html_header_show() {
     //  We commented this out since we are now standardizing encoding
     //  in the globals.php file.
```

Comment:
Could migrate this function to a library file (perhaps library/header_footer_global.php) and then include it in globals.php:
https://github.com/openemr/openemr/blob/master/interface/globals.php#L158-L174